### ISSUE TITLE
docker hub updater is down :(

### ISSUE THREAD

**Issue by imzenreally** [factoriotools/factorio-docker#431]
dunno where else to leave a comment - github probably isn't the right place but dockerhub is still hosting 1.56

**Comment by bplein** [comment]
It appears that this repo has not yet been upgraded to 1.1.57. Docker will move forward AFTER the repo is updated. @Fank usually submits the changes, sometimes things break but it will be on track soon enough.

**Comment by SuperSandro2000** [comment]
Yeah, we could use github actions to generate the PRs. Its just someone needs to find time and do it.
